SimTracker/TrackerMaterialAnalysis/test/listIds.py

```diff
@@ -1,22 +0,0 @@
-#! /usr/bin/env cmsRun
-
-import FWCore.ParameterSet.Config as cms
-
-process = cms.Process("MaterialAnalyser")
-process.load("Configuration.StandardSequences.Geometry_cff")
```

Comment:
I think the reason why this was not working is that `Configuration.StandardSequences.Geometry_cff` does not exit (anymore).
Have you tried adding:
```
process.load('Configuration.Geometry.GeometryDB_cff')
process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_cff')
from Configuration.AlCa.GlobalTag import GlobalTag
process.GlobalTag = GlobalTag(process.GlobalTag, 'auto:run2_mc', '')
```

?

Reply:
File restored, DB added, it runs now